release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -238,6 +238,68 @@ In the table, features are marked with the following statuses:
 [id="ocp-4-9-removed-features"]
 === Removed features
 
+==== Beta APIs removed from Kubernetes v1.22
+
+The upgrade to Kubernetes v1.22 removed the following deprecated
```

Comment:
I'm not sure I would say "the upgrade to", not sure if that's really correct. I'd just say "Kubernetes 1.22 removed..."